src-graphql/client.js

```diff
@@ -87,4 +87,67 @@ export default class Client {
       return response.model.node;
     });
   }
+
+  fetchQueryProducts(queryObject, query = productConnectionQuery()) {
```

Comment:
Default here? `queryObject = {}`. Should eliminate the need for the `queryObject` conditional.